SCloudRandomizer/MainViewController.m

```diff
@@ -429,10 +430,13 @@ - (void)setupLockScreenInfo:(NSDictionary *)track
         NSData *imageData = [NSData dataWithContentsOfURL:imgUrl];
         MPMediaItemArtwork *albumArtwork = [[MPMediaItemArtwork alloc] initWithImage:[UIImage imageWithData:imageData]];
         
-        NSArray *keys = [NSArray arrayWithObjects:MPMediaItemPropertyAlbumTitle, MPMediaItemPropertyArtist, MPMediaItemPropertyArtwork, nil];
-        NSArray *values = [NSArray arrayWithObjects:trackArtist, trackTitle, albumArtwork, nil];
+        NSArray *keys = [NSArray arrayWithObjects:MPMediaItemPropertyArtist, MPMediaItemPropertyTitle, MPMediaItemPropertyArtwork, MPMediaItemPropertyPlaybackDuration, MPNowPlayingInfoPropertyPlaybackRate, nil];
+        NSArray *values = [NSArray arrayWithObjects:trackArtist, trackTitle, albumArtwork, actualDuration, [NSNumber numberWithInt:1], nil];
```

Comment:
Yes its like sort like strings in C, you need to null (nil) terminate. Its used when you are iterating through it.